assets/scss/base/_grid.scss

```diff
@@ -52,6 +53,14 @@ Styleguide Grids
   }
 }
 
+.grid-layout--half {
```

Comment:
this feels like it could end up being: 
```
.grid-layout--maintain-width {
 @include media(mobile) {
  .grid-layout__column--1-2 {
    width: $half;
  }
  .grid-layout__column--1-4 {
    width: $one-quarter;
  }
 .grid-layout__column--3-4 {
    width: $three-quarters;
  }
 .grid-layout__column--1-3 {
    width: $one-third;
  }
 .grid-layout__column--2-3 {
    width: $two-thirds;
  }
 }
}
```